ts/src/okx.ts

```diff
@@ -2417,6 +2418,8 @@ export default class okx extends Exchange {
         const timezone = this.safeString (options, 'timezone', 'UTC');
         if (limit === undefined) {
             limit = 100; // default 100, max 100
+        } else {
+            limit = Math.min (limit, 300); // max 100
```

Comment:
Just to clarify, is the maximum limit 100 or 300?

Reply:
![image](https://github.com/user-attachments/assets/c7bec513-d58c-44b9-8925-6b9b425c1a52)
for the regular endpoint 300, the historical 100
![image](https://github.com/user-attachments/assets/370080ec-177a-4926-b539-3091a0416f8d)